test/testutil/outcome.hpp

```diff
@@ -27,11 +27,11 @@
 
 #define EXPECT_OUTCOME_FALSE_void(var, expr) \
   auto &&var = expr;                         \
-  EXPECT_FALSE(var) << "Line " << __LINE__ << ": " << var.error().message();
```

Comment:
Because when var does have a value, var.error() throws an exception with description "no error", which looks cryptic. Without this it correctly prints that var was expected to be false, but is actually true.